tests/testthat/test-cpp.R

```diff
@@ -1,3 +1,3 @@
-test_that("Catch unit tests pass", {
-  expect_cpp_tests_pass("testthat")
-})
+#test_that("Catch unit tests pass", {
```

Comment:
With this new style we would get duplicate results if we run it within a `test_that()` block